cmd/soroban-rpc/internal/test/integration.go

```diff
@@ -35,10 +35,11 @@ import (
 
 const (
 	StandaloneNetworkPassphrase = "Standalone Network ; February 2017"
-	stellarCoreProtocolVersion  = 20
+	StellarCoreProtocolVersion  = 20
```

Comment:
any particular reason that we should export this const ?
given that it's being used as part of a test, I think that we should keep it as "local" as possible.

Reply:
Yep, good catch. I was thinking it was a separate test package, but it is not. Will fix.